Code/Editor/EditorFramework/CodeGen/CppProject.cpp

```diff
@@ -515,3 +901,241 @@ bool ezCppProject::IsBuildRequired()
 
   return false;
 }
+
+namespace
+{
+  template <typename T>
+  T* Expect(ezVariantDictionary& inout_json, ezStringView sName)
```

Comment:
that's quite useful, maybe we should add something like this directly to ezVariant

Reply:
Let's discuss this and I'll do it as a follow up PR